src/signals/incident/containers/IncidentContainer/reducer.js

```diff
@@ -13,44 +13,49 @@ import {
   CREATE_INCIDENT_ERROR,
 
   GET_CLASSIFICATION_SUCCESS,
-  GET_CLASSIFICATION_ERROR
+  GET_CLASSIFICATION_ERROR,
+
+  SET_PRIORITY,
+  SET_PRIORITY_SUCCESS,
+  SET_PRIORITY_ERROR
 } from './constants';
 
 export const initialState = fromJS({
   incident: {
-    // location: {
-    // address: {
-    // openbare_ruimte: 'Wilhelminastraat',
-    // huisnummer: 42,
-    // huisletter: '',
-    // huisnummer_toevoeging: '1',
-    // postcode: '1054WJ',
-    // woonplaats: 'Amsterdam'
-    // },
-    // buurt_code: 'E20c',
-    // stadsdeel: 'E',
-    // geometrie: {
-    // type: 'Point',
-    // coordinates: [
-    // 52.36223071695314,
-    // 4.865698814392091
-    // ]
-    // }
-    // },
-    // subcategory: 'Overlast op het water - snel varen',
-    // category: 'Overlast op het water',
-    // description: 'snel boot',
-    // description: 'poep',
-    // phone: '020654321',
-    // email: 'a@b.com',
-    // extra_boten_snelheid_rondvaartboot: 'Ja',
-    // extra_boten_snelheid_rederij: 'Aemstelland',
-    // datetime: 'Nu',
+    location: {
```

Comment:
This is not a correct initial state I believe

Reply:
sorry

Reply:
I'd prefer it if this test initial state is at least moved to a separate file